tests/test_layers.py

```diff
@@ -0,0 +1,49 @@
+import geopandas as gpd
+import pytest
+from shapely.geometry import MultiPolygon, Polygon
+
+from gpbp.layers import AdmArea
+
+
+@pytest.fixture
+def multipolygon():
+    # Create two simple square polygons
+    polygon1 = Polygon([(0, 0), (1, 0), (1, 1), (0, 1)])
+    polygon2 = Polygon([(2, 2), (3, 2), (3, 3), (2, 3)])
+
+    # Combine them into a MultiPolygon
+    return MultiPolygon([polygon1, polygon2])
```

Comment:
@dcossao I refactored the fixtures so there's only one now, which is more useful than before.